sample.env

```diff
@@ -1,3 +1,12 @@
 PG_USER=admin
 PG_PW=admin
 PG_DB=beyond_life
+BACKEND_DB_USER_NAME=admin
+BACKEND_DB_USER_PW=admin
+BACKEND_DB_DB=beyond_life
```

Comment:
While in theory, yes, in practice I would prefer staying true to the env vars of the service. Otherwise any changes could lead to unintended side effects